public/coffee/ide/review-panel/controllers/ReviewPanelController.coffee

```diff
@@ -27,6 +28,16 @@ define [
 			rendererData: {}
 			loadingThreads: false
 
+		$scope.$on "project:joined", () ->
+			$scope.reviewPanel.overview.docsCollapsedState = JSON.parse(localStorage("docs_collapsed_state:#{$scope.project_id}")) or {}
```

Comment:
Doing this on join project will reset it if the client loses connection and reconnects. Just do it once on load, i.e. when the controller is created?